docs/developers/deploy-solidity-ethermint-foundry.md

```diff
@@ -0,0 +1,229 @@
+# Deploy A Solidity Smart Contract on Ethermint Sovereign Rollup with Foundry
+
+In this guide you'll learn how to deploy a Solidity smart contract to
+the Ethermint chain you just instantiated on Celestia with
+[Foundry](https://github.com/foundry-rs/foundry).
+
+## About Foundry
+
+Foundry is a portable, fast and modular toolkit for Ethereum application development.
+
+Foundry is made up of three components:
+
+- [__Forge__](https://github.com/foundry-rs/foundry/tree/master/forge): Ethereum
+ testing framework (like Truffle, Hardhat and DappTools).
+- [__Cast__](https://github.com/foundry-rs/foundry/tree/master/cast): CLI for
+interacting with EVM smart contracts, sending transactions, and getting chain data.
+- [__Anvil__](https://github.com/foundry-rs/foundry/tree/master/anvil): Local
+Ethereum node, similar to Ganache or Hardhat Network.
+
+We'll use all three to create, test, and deploy our Solidity project.
+
+> To learn more about Foundry, check out the [Foundry Book](https://book.getfoundry.sh/).
+
+## Getting started
+
+### Initialize development environment
+
+First, be sure to
+[install Foundry](https://book.getfoundry.sh/getting-started/installation.html)
+on your local development environment.
+
+Next, create a new project and change into the directory:
+
+```sh
+forge init celestia-ethermint-app
```

Comment:
hmm..what am I missing here? running in wrong directory? need to initialize git directory first?
```
root@jcs:~# forge init celestia-ethermint-app
Initializing /root/celestia-ethermint-app...
Installing forge-std in "/root/celestia-ethermint-app/lib/forge-std" (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
Error:
fatal: not a git repository (or any of the parent directories): .git
```

Reply:
we found it to be a bug in foundry but they have a fix which will come out in a new release. error doesn't block on initializing project.